translations/ko.po

```diff
@@ -248,17 +248,20 @@ msgstr "일치하는 이름을 Yugipedia 페이지에서 찾을 수 없습니다
 msgid "${ counts.Monster } Monster"
 msgid_plural "${ counts.Monster } Monsters"
 msgstr[0] ""
+msgstr[1] ""
```

Comment:
I haven't figured out where this bug is in ttag yet, but it unnecessarily adds these only when updating and the entry already exists. When the entry already exists, it won't create these for languages that don't have plural forms.